Doc/library/fcntl.rst

```diff
@@ -155,6 +155,12 @@ The module defines the following functions:
    The default for *len* is 0 which means to lock to the end of the file.  The
    default for *whence* is also 0.
 
+   The default for *open_file_descriptor* is False, if *open_file_descriptor* is
+   set to True, open file description locks are used.
```

Comment:
This is cryptic. Few people know what an "open file description lock" is. Can you perhaps link to a manual page or some kind of doc on the topic?

Reply:
Also, if this is Linux-specific, then it should also be mentioned (e.g. "open file description locks are a Linux-specific feature").

Reply:
@pitrou I agree